Venmo/src/main/java/com/braintreepayments/api/venmo/VenmoPaymentAuthResultInfo.kt

```diff
@@ -0,0 +1,13 @@
+package com.braintreepayments.api.venmo
+
+import com.braintreepayments.api.BrowserSwitchFinalResult
+
+/**
+ * Details of a [VenmoPaymentAuthResult.Success]
+ */
+class VenmoPaymentAuthResultInfo internal constructor
```

Comment:
Same here - this should follow the same format as PayPal: https://github.com/braintree/braintree_android/blob/main/PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalPaymentAuthResultInfo.kt